docs/docs/agents/agents.md

```diff
@@ -85,12 +92,33 @@ agent.invoke(
 
 See the [context](./context.md) page for more information.
 
+### Static prompts
```

Comment:
should it go first then? (before dynamic?)

Reply:
yes it should, feels a bit like duplication -- we might end up figuring out how to paraphrase and remove this part